Makefile

```diff
@@ -1,6 +1,6 @@
-PRODUCT_MAJOR_VERSION=7
-PRODUCT_MINOR_VERSION=31
-PRODUCT_MICRO_VERSION=0
+PRODUCT_MAJOR_VERSION=8
+PRODUCT_MINOR_VERSION=2
+PRODUCT_MICRO_VERSION=1
```

Comment:
See comments on https://github.com/xenserver/xe-guest-utilities/pull/157

Reply:
There is different version number with different branch. 8.2.1 is for Yangtze and 8.3.1 is for Next

Reply:
updated